cmd/starter/start.go

```diff
@@ -68,8 +78,9 @@ func startHelper(cmd *cobra.Command, args []string) {
 
 	select {
 	case <-ctx.Done():
-		return
-	case err := <-errorPipeline:
-		log.Fatal(ctx, err.Error())
+		return nil
+	case errPipeline := <-errorPipeline:
```

Comment:
nit: "errPipeline" -> "err"

Reply:
Is your suggestion to make it `case err := <-errorPipeline` or case `errPipeline := <-err `
I had to change from err to errPipeline since once of the lint errors was that the old err was being overshadowed by an err above.